salt/modules/htpasswd.py

```diff
@@ -139,3 +139,46 @@ def userdel(pwfile, user, runas=None):
     out = __salt__['cmd.run'](cmd, runas=runas,
                               python_shell=False).splitlines()
     return out
+
+
+def verify(pwfile, user, password, opts='', runas=None):
+    '''
+    Return True if the htpasswd file exists, the user has an entry, and their
+    password matches.
+
+    pwfile
+        Path to htpasswd file
+
+    user
+        User name
+
+    password
+        User password
+
+    opts
+        Valid options that can be passed are:
```

Comment:
After looking at this more closely, I think your argument for keeping it as-is is reasonable.